pyroute2/ipdb/routes.py

```diff
@@ -608,17 +616,25 @@ def make_key(cls, msg):
         values = []
         if isinstance(msg, nlmsg_base):
             for field in RouteKey._fields:
+                # only IPv4 uses 'tos' as part of the RouteKey
+                if field == 'tos' and \
+                        msg.get('family', None) != AF_INET:
+                  continue
```

Comment:
indentation is not a multiple of four (E111)

Reply:
Oops!  Will fix.

Reply:
I use flake8 (https://pypi.python.org/pypi/flake8) to catch such things. Actually, even the flake8 plugin to the vim (https://github.com/nvie/vim-pep8). Much easier when it backs me up.

Reply:
Thanks!  I'll try that.